src/components/common/bullets-colin-link.vue

```diff
@@ -0,0 +1,212 @@
+<template>
+  <div id="bullets-colin-link">
```

Comment:
You probably don't need this div, and can use v-row as the main element in this template.